parser/parser/cc/token.cc

```diff
@@ -1,4 +1,5 @@
 #include <cassert>
+#include <iostream>
```

Comment:
This looks unused?
```suggestion
```

Reply:
It's used by the printing code below (there is a definition of `operator<<`.

It used to get this include for free because the order of the header file includes.